plugins/discourse-local-dates/test/javascripts/acceptance/local-dates-composer-test.js

```diff
@@ -128,4 +131,26 @@ acceptance("Local Dates - composer", function (needs) {
     await click("ul.formats a.moment-format");
     assert.dom("input.format-input").hasValue("LLL");
   });
+
+  test("composer insert current time shortcut", async function (assert) {
+    await visit("/t/internationalization-localization/280");
+
+    await click("#topic-footer-buttons .btn.create");
+    assert.dom(".d-editor-input").exists("the composer input is visible");
```

Comment:
also nitpick, but this line to me is useless. You do `fillIn(".d-editor-input")` on next line, that will fail if the input is not here.